### ISSUE TITLE
Debug utPLSQL test

### ISSUE THREAD

**Issue by PhilippSalvisberg** [utPLSQL/utPLSQL-SQLDeveloper#87]
# Current Behavior

In version 1.1.0 you can run utPLSQL tests. But if you want to debug a utPL/SQL test, some manual steps are involved. There are several ways to do that. 

One way is to run a utPLSQL test in a worksheet change the code to a simple anonymous PL/SQL block (no SQL*Plus statements are allowed) to enable the `Debug...` context menu option and then select this menu option to launch the debugger. 

### 1. Launch Debugger

![utplsql_1_launch_debugger](https://user-images.githubusercontent.com/3883171/63647366-92f63280-c720-11e9-9c70-de81b81e83f4.png)

### 2. Debugging Session

![utplsql_2_debugging_session](https://user-images.githubusercontent.com/3883171/63647381-c1740d80-c720-11e9-8064-8ac41ce9dcb9.png)

# Expected Behavior

## Launch the Debugger directly

1. From the context menu of the Database Connections navigator tree, for one or multiple selected nodes for which a "Run utPLSQL test" menu item is available in v1.1.0.

2. From the Worksheet or PL/SQL editor via context menu, when the "Run utPLSQL test" menu item is available in v1.1.0.

3. From the Realtime Reporter via toolbar.

4. From the Realtime Reporter via context menu on the overview table for the selected test(s).

## Out of Scope

The following is not in scope of the utPLSQL for SQL Developer extension. At least they are not considered part of this issue.

### Managing Breakpoints

Breakpoints need to be set in advance or during the debugging session.  

### Compiling PL/SQL Code for Debug

This needs to be done before starting a debug session. Either for chosen components via SQL Developer or via a script recompiling all relevant code for debug.
